core/privc/privc_context.h

```diff
@@ -43,12 +47,18 @@ class PrivCContext : public AbstractContext {
 
   PrivCContext &operator=(const PrivCContext &other) = delete;
 
+  void set_triplet_generator(std::shared_ptr<TripletGenerator<int64_t, SCALING_N>>& tripletor);
```

Comment:
do we need to replace the tripliet_generator during using? if not, no need the setter.